pkg/statdb/statdb.go

```diff
@@ -178,6 +178,7 @@ func (s *Server) Update(ctx context.Context, updateReq *pb.UpdateRequest) (resp
 
 // UpdateBatch for updating  multiple farmers' stats in the db
 func (s *Server) UpdateBatch(ctx context.Context, updateBatchReq *pb.UpdateBatchRequest) (resp *pb.UpdateBatchResponse, err error) {
+	// todo(moby) how should we handle one node failing to update but not all?
```

Comment:
Missing
var mon = monkit.Package() and every function should have
defer mon.Task()(&ctx)&(err) for profiling and debugging purposes